metrics/wer/README.md

```diff
@@ -0,0 +1,118 @@
+# Metric Card for WER
+
+## Metric description
+Word error rate (WER) is a common metric of the performance of an automatic speech recognition (ASR) system. 
+
+The general difficulty of measuring performance lies in the fact that the recognized word sequence can have a different length from the reference word sequence (supposedly the correct one). The WER is derived from the [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance), working at the word level instead of the phoneme level. 
+
+This problem is solved by first aligning the recognized word sequence with the reference (spoken) word sequence using dynamic string alignment. Examination of this issue is seen through a theory called the power law that states the correlation between [perplexity](https://huggingface.co/metrics/perplexity) and word error rate.
```

Comment:
```suggestion
The WER is computed by first aligning the recognized word sequence with the reference word sequence using dynamic word alignment. 

Theory on the meaningfulness of WER is closely linked to the so-called power law that states the correlation between [perplexity](https://huggingface.co/metrics/perplexity) and word error rate.
```

Do you think we could maybe add a link (wikipedia, article, paper) to the last sentence so that the reader - if interested - could take a closer look at the link between WER and perplexity?